llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -7362,6 +7363,18 @@ AArch64TargetLowering::LowerCall(CallLoweringInfo &CLI,
   else if (auto *ES = dyn_cast<ExternalSymbolSDNode>(CLI.Callee))
     CalleeAttrs = SMEAttrs(ES->getSymbol());
 
+  auto DescribeCallsite =
+      [&](OptimizationRemarkAnalysis &R) -> OptimizationRemarkAnalysis & {
+    R << "call from " << ore::NV("Caller", MF.getName()) << " to ";
```

Comment:
Can you add single quotes around the function names? That's what other remarks seem to do (such as the inliner)